docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-model/creating-vehicle-model/index.md

```diff
@@ -0,0 +1,274 @@
+# Creating a vehicle model for Autoware
+
+## Introduction
+
+This page introduces the following packages for the vehicle model:
+
+1. `<YOUR-VEHICLE-NAME>_vehicle_description`
+2. `<YOUR-VEHICLE-NAME>_vehicle_launch`
+
+So,
+we forked our vehicle model
+at [creating autoware repositories](../../creating-your-autoware-meta-repository/creating-autoware-meta-repository.md) page step,
+(For example,
+we created [tutorial_vehicle_launch](https://github.com/leo-drive/tutorial_vehicle_launch) for our documentation vehicle at this step)
+please be sure `<YOUR-VEHICLE-NAME>_vehicle_launch` repository is included in Autoware like the following directory:
+
+```diff
+<YOUR-OWN-AUTOWARE-DIR>/
+  └─ src/
+       └─ vehicle/
+            └─ <YOUR-VEHICLE-NAME>_vehicle_launch/
+                 ├─ <YOUR-VEHICLE-NAME>_vehicle_description/
+                 └─ <YOUR-VEHICLE-NAME>_vehicle_launch/
+```
+
+If your forked repository doesn't include in the correct structure like above,
+please add your forked sensor_kit repo to autoware.repos file
+and run `vcs import src < autoware.repos` command on your terminal
+to import new included repositories at autoware.repos file.
```

Comment:
@ismetatabay I am a little confused by these lines. I believe this is what you meant?

```suggestion
If your forked Autoware meta-repository doesn't include `<YOUR-VEHICLE-NAME>_vehicle_launch` with the correct folder structure as shown above, please add your forked `<YOUR-VEHICLE-NAME>_vehicle_launch` repository to the autoware.repos file and run the vcs import src < autoware.repos command in your terminal to import the newly included repositories at autoware.repos file.
```

Reply:
Thank you for your suggestion. Yes, that is what i mean.